.yamllint.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+rules:
+  indentation:
+    spaces: 2
+    indent-sequences: whatever
```

Comment:
docs, here, https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation
see https://github.com/adrienverge/yamllint/issues/80 for an example of the default behavior
it would have wanted 
```
python:
  - changed-files:
      - any-glob-to-any-file:
          - '**.py'
          - '**/poetry.lock'
          - '**/pyproject.toml'
          - '**/requirements.txt'
```

I'm not sold one way or the other, I go off of common text reading/writting convention of the first thing on the line being the indicator of scope so I gravitate toward the `-` being included in the indent.